ttkwidgets/numberentry.py

```diff
@@ -0,0 +1,91 @@
+"""
+Author: rdbende
+License: GNU GPLv3
+Copyright (c) 2021 rdbende
+"""
+
+import tkinter as tk
+from tkinter import ttk
+
+
+class NumberEntry(ttk.Entry):
+    """
+    An entry that takes only numbers or calculations and calculates the result of the calculation
+    
+    :param expressions: Allow the use of expressions (default is True)
+    :type expressions: bool
+    
+    :param roundto: The number of decimals in the result (default is 0)
+    :type roundto: int
+    """
+    
+    
+    def __init__(self, master, **kwargs):
+        
+        
+        self._expr = kwargs.pop("expressions", True)
+        self._round = kwargs.pop("roundto", 0)
+        ttk.Entry.__init__(self, master, **kwargs)
+        self.bind("<Return>", self._eval)
+        self.bind("<FocusOut>", self._eval)
+        self.bind("<KeyRelease>", self._check)
+    
+    
```

Comment:
[PEP-8](https://www.python.org/dev/peps/pep-0008/#blank-lines) specifies only a single blank line ahead of a class method.

Reply:
I still can't get used to PEP-8, but I fixed it